libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -43,8 +43,7 @@ class SRV_Channel {
 
     static const struct AP_Param::GroupInfo var_info[];
 
-    typedef enum
-    {
+    enum class Function : uin8_t {
```

Comment:
needs to be signed, and needs to be a real type

Reply:
This was a bad patch, now removed